go/service/kvstore_test.go

```diff
@@ -743,8 +743,7 @@ func TestManualControlOfRevisionWithoutCache(t *testing.T) {
 	require.Equal(t, delRes.Revision, 3)
 }
 
-// TestKVStoreLocalRace tests that multiple requests at the same time are caught by the server
```

Comment:
is it worth testing that multiple requests at the same time from multiple clients (as in, multiple handlers with distinct local revision cache instances) doesn't cause unexpected errors?

Reply:
k. done.